.github/workflows/test.yml

```diff
@@ -21,10 +21,16 @@ jobs:
         with:
           profile: minimal
           toolchain: ${{ matrix.toolchain }}
+          target: wasm32-unknown-unknown
+      - name: downgrade `anstyle`, `clap`, `clap_lex` crates to support older Rust toolchain
+        if: matrix.toolchain == '1.69.0' 
+        run: |
+          cargo update -p anstyle --precise 1.0.2
+          cargo update -p clap --precise 4.3.24
+          cargo update -p clap_lex --precise 0.5.0
       - uses: Swatinem/rust-cache@v1
-      - run: rustup target add wasm32-unknown-unknown
       - name: Test
-        run: cargo test --all --features unstable,legacy
+        run: cargo +${{ matrix.toolchain }} test --all --features unstable,legacy
```

Comment:
Well, instead of manually specifying +version, let’s set it as the default toolchain in the `uses: actions-rs/toolchain@v1` block